### DISCUSSION TITLE
Rails 7 esbuild with taildwind CSS not found working solution.

### DISCUSSION THREAD

**Discussion by vinayrails-tc** [activeadmin/activeadmin#8226] [Help]
Ruby version : 3.1.3
Rails version : 7.0.4

project generate command : rails new project_name -j esbuild --css tailwind

I've tried many things but none of can working now.
Is there any possible way to add tailwind css into activeadmin.
Please help me out in this.

**Comment by tagliala** [comment]
Hi, please take a look here: https://github.com/activeadmin/activeadmin/issues/7264

The main problem is that jquery-ui does not play well with esbuild

You may also be interested in version 4.0.0.beta3, which uses tailwind and drops jquery

**Reply by vinayrails-tc** [reply]
Thank you, @tagliala for the quick response.

Can you please share the steps to install rails 7 with tailwind?
